README.md

```diff
@@ -1,24 +1,49 @@
-# Kailua
+# OP KAILUA
+<img width="1280" alt="Github_cover page (3)" src="https://github.com/user-attachments/assets/c71a1018-d970-49c0-af37-ba8a66d00bea" />
```

Comment:
The link preview will show this html text if we put it at the very top. 
I suggest putting a short description before the image in order to avoid a messy link preview.